src/main/resources/mybatis/discreteGWWaterMLPoint.xml

```diff
@@ -20,11 +24,16 @@
 			approval_level_description "status"
 		from nwis.discrete_ground_water_aqts dgw
 		join nwis.monitoring_location ml
-		on ml.monitoring_location_identifier = ml.monitoring_location_identifier
+		on dgw.monitoring_location_identifier = ml.monitoring_location_identifier
 		left join nwis.parm parm
 			on dgw.parameter_code = parm.parm_cd
 		where
 			dgw.monitoring_location_identifier = #{featureId,jdbcType=VARCHAR}
-		order by "featureId", "pcode", "statisticDesc", "resultDateTimeUTC"
+		WINDOW wnd AS (
+			PARTITION BY dgw.monitoring_location_identifier, parameter_code
```

Comment:
monitoring_location_identifier not really required here.

Reply:
...kept in anticipation of accepting multiple monitoring locations in the query.